myuw/management/commands/update_past_instructors.py

```diff
@@ -0,0 +1,33 @@
+from django.core.management.base import BaseCommand
+from uw_sws.section import get_changed_sections_by_term, get_section_by_url
+from uw_sws.term import get_current_term, get_term_before
+from myuw.models import User, SeenInstructor
+from myuw.dao.instructor_schedule import get_prior_instructed_terms
+
+
+class Command(BaseCommand):
+    help = "Add instructors from past quarters to the SeenInstructor table"
```

Comment:
Could you add a dao.instructor.py with functions like:  add_seen_instructor and purge_seen_instructors ?
Also add unti tests.